app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    try:
+        assert len(command.split()) == 3, "Command should have 3 arguments"
+        assert command.split()[0] == "mv", "First command should be 'mv'"
+    except AssertionError as e:
+        print(e)
+        return
+
+    order, old_file, new_location = command.split()
```

Comment:
Better to put in to the begging of the function and use in this assert `assert command.split()[0] == "mv"`